release/pkg/assets/config/bundle_release.go

```diff
@@ -813,6 +815,32 @@ var bundleReleaseAssetsConfigMap = []assettypes.AssetConfig{
 	},
 }
 
-func GetBundleReleaseAssetsConfigMap() []assettypes.AssetConfig {
-	return bundleReleaseAssetsConfigMap
+func GetBundleReleaseAssetsConfigMap(rc *releasetypes.ReleaseConfig) []assettypes.AssetConfig {
+	conf := bundleReleaseAssetsConfigMap
+	if rc.DevRelease && rc.BuildRepoBranchName == "main" {
```

Comment:
This check is performed in assets/assets.go, and since we set `OnlyForDevRelease` to `true`, this AssetConfig can just be added to the above list.

Reply:
If you take a look at `TestGenerateBundleManifest`,[ the release config](https://github.com/aws/eks-anywhere/blob/15a1cd2a1229abece283f24a2b499ee577b96006/release/pkg/operations/bundle_release_test.go#L59) has `DevRelease` set to true for both `main` and `release-0.11` branch. As such, it tries to fetch a GIT tag for a repo that doesn't exist and fails.

Reply:
Commenting out `release-0.11` test case for now.